<launch>
     <node pkg="rosbag" type="record" name="robot1_record" 
         args="record -O ../catkin_ws/src/comunicating_robots/bagfiles/robot1_bag.bag stateTopic_robot1 "/>  

     <node pkg="rosbag" type="record" name="w_robot1_record"
         args="record -O ../catkin_ws/src/comunicating_robots/bagfiles/w_robot1_bag.bag inputTopic_robot1"  />  

     <node pkg="rosbag" type="record" name="var_robot1_record" 
         args="record -O ../catkin_ws/src/comunicating_robots/bagfiles/var_robot1_bag.bag varTopic_robot1" /> 

     <arg name="robot_name1"/>
     <node pkg="comunicating_robots" ns="sim1" name="simulator_robot1" type="simulator.py" output="screen" />
        <param name="sim1/" value="$(arg robot_name1)"/>

  <node pkg="comunicating_robots" ns="sim1" name="pdes_comunicator_robot1" type="pdes_comunicator.py" />

  <node pkg="comunicating_robots" ns="sim1" name="controller_robot1" type="controller.py" output="screen" launch-prefix="gnome-terminal --command"/>



</launch>
